lib/routes/gov/moa/moa.ts

```diff
@@ -5,27 +5,34 @@ import { load } from 'cheerio';
 import { parseRelativeDate } from '@/utils/parse-date';
 
 const hostUrl = 'http://www.moa.gov.cn/';
-const hostUrlObj = new URL(hostUrl); // 用于在下面判断host
+const hostUrlObj = new URL(hostUrl); // 用于在下面判断 host
 
 export const route: Route = {
     path: '/moa/:suburl{.+}',
+    categories: ['government'],
+    example: '/gov/moa/gk/zcjd/',
     radar: [
         {
             source: ['moa.gov.cn/'],
             target: '/moa/:suburl',
         },
     ],
-    name: 'Unknown',
-    maintainers: [],
+    parameters: { subrul: '下级目录，请使用最下级的目录' },
```

Comment:
```suggestion
    parameters: { suburl: '下级目录，请使用最下级的目录' },
```
https://github.com/DIYgod/RSSHub/pull/15012#discussion_r1548112696